examples/example.tf

```diff
@@ -3,10 +3,25 @@ provider "aws" {
   region  = "us-west-2"
 }
 
+provider "aws" {
+  alias  = "aws_n_va"
+  region = "us-east-1"
+}
+
+data "aws_route53_zone" "zone" {
+  provider = aws.aws_n_va
```

Comment:
maybe not needed?

Reply:
@jvisker done I think. Want to double check?

Reply:
Looks good